source/_patterns/02-molecules/blocks/alert/alert.js

```diff
@@ -1,10 +1,54 @@
-function alertCloseclick() {
-  document.getElementById("alert-close").onclick = function alertClose() {
-    var alert = document.getElementById("alert-bar");
-    if (alert.style.display === "none") {
-      alert.style.display = "block";
-    } else {
-      alert.style.display = "none";
+let messageContentObject = document.getElementsByClassName("usa-alert__text"),  cookieId;
+if(messageContentObject[0]) {
+  cookieId = stringToHash(messageContentObject[0].innerText);
```

Comment:
@baptisten let's add a prefix to the cookieID, so it's easy to spot in storage, something like "patternlab-alert-hide-"